changelog.md

```diff
@@ -95,6 +95,9 @@
 - [#3701](https://github.com/ignite/cli/pull/3701) Bump `go` version to 1.21
 
 ## [`v0.27.1`](https://github.com/ignite/cli/releases/tag/v0.27.1)
+- [#3869](https://github.com/ignite/cli/pull/3869) Fix .git in parent dir
```

Comment:
the changelog should be under unreleased fixes.

Reply:
Sorry. Git auto merge goes wrong.